cookies/models.py

```diff
@@ -53,8 +53,16 @@ class Entity(models.Model):
     A named object that represents some element in the data.
     """
 
+    INTERFACE_WEB = 'Web'
```

Comment:
It's nitpicking, but can we have "Web" in all uppercase letters please. To make clear it's a key and not meant to be used for anything else.